frontend/src/views/Market.vue

```diff
@@ -700,6 +700,23 @@
         </div>
       </b-tab>
     </b-tabs>
+    <b-modal class="centered-modal text-center" ref="character-buy-modal" :title="$t('market.characterBuyModal')"
+      @ok="purchaseNft" @cancel="purchaseNft(true)">
```

Comment:
On cancel call purchaseNft?

Reply:
Yes, there's no cancel option, but I'm using the cancel handler event.